.github/workflows/ci.yml

```diff
@@ -39,7 +24,7 @@ jobs:
         swift-action: 'test'
 
   validate-podspec:
```

Comment:
to get this working we'll likely need to bump up CocoaPods in the Gemfile

Reply:
You're right -- the job passes now after updating to the latest version of cocoapods